docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -495,6 +496,99 @@ We assume family history does not affect incidence rates from LCIS to breast can
  - treatment model baseline (using GBD incidence)
  - screening scale-up and treatment coverage scale-up model (changes incidence, should reduce breast cancer prevalence, mortaliaty and morbidity)
 
+:underline:`Baseleine scenario for DCIS`
+
+ :math:`i_{BC|DICS}  = i_{BC|DCIS{tx1}} \times \ P_{tx1} + i_{BC|DCIS{tx0}} \times \ (1-P_{tx1})` 
+
+
+- :math:`i_{BC|DICS}` is the incidence of breast cancer from DCIS (baseline) based on GBD's i_BC
+- let :math:`P_{tx1}` be the proportion of people who have DCIS and treatment. 
+- :math:`1- P_{tx1}` is the poportion of people who have DCIS and no treatment
+- let :math:`i_{BC|DCIS{tx1}}` be the incidence of breast cancer from DCIS after treatment.
+- let  :math:`i_{BC|DCIS{tx0}}` be the incidence of breast cancer from DCIS among those without treatment (we can caclulate this based on the equation above and apply to scale up scenario). 
+
+
+(1) :math:`i_{BC|DCIS{tx1}}` = 882 (95%CI 845-921) per 100,000 person years (Mannu 2020 **BMJ**)
```

Comment:
I wonder how this compares to the GBD value.  It is possible that there is not a consistent way to solve for `i_{BC|DCIS{tx0}}`.  I guess we shall see.